packages/api-bindings/src/lens/common.graphql

```diff
@@ -204,11 +204,43 @@ fragment Media on Media {
   url
 }
 
-fragment MediaSet on MediaSet {
+fragment PublicationMediaSet on MediaSet {
   __typename
   original {
     ...Media
   }
+  optimized {
+    ...Media
+  }
+  small: transformed(params: $mediaTransformPublicationSmall) {
+    ...Media
+  }
+  medium: transformed(params: $mediaTransformPublicationMedium) {
+    ...Media
+  }
+}
+
+fragment ProfilePicture on MediaSet {
+  __typename
+  original {
+    ...Media
+  }
+  optimized {
+    ...Media
+  }
+  thumbnail: transformed(params: $mediaTransformProfileThumbnail) {
+    ...Media
+  }
+}
+
+fragment ProfileCoverPicture on MediaSet {
```

Comment:
introduce new fragments for PublicationMediaSet, ProfilePicture and ProfileCoverPicture as they require different size transformations

Reply:
Naming wise there seems to be some inconsistencies in the use of the "set" suffix.

Maybe:

- `PublicationMediaSet` (unchanged) - I think `Media` correctly carry the fact it't not necessarily an image.
- `ProfilePictureSet` - `PictureSet` like if it were to be a specialized type of `MediaSet`, a "set of pictures"
- `ProfileCoverPictureSet` - same as above